tests/integration-v1/cattletest/core/test_container.py

```diff
@@ -517,12 +517,16 @@ def test_container_logs(context):
 
 
 def test_container_labels(client, context):
-    labels = {'affinity': "container==B", '!affinity': "container==C"}
+    labels = {'affinity': "container==B", '!affinity': "container==C",
+              'io.rancher.environment.uuid': "default"}
     container = context.create_container(name="test" + random_str(),
                                          labels=labels)
     container = client.wait_success(container)
     assert container.state == 'running'
-    assert container.labels == labels
+    assert container.labels['affinity'] == "container==B"
+    assert container.labels['!affinity'] == "container==C"
+    key = 'io.rancher.environment.uuid'
```

Comment:
Nit: you can define the key at the beginning of the method. This way you can reuse it in the labels definition